### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by candidexmedia**
Added more instructions, based on berndmj's comment: https://www.reddit.com/r/kdenlive/comments/12p8ryh/comment/jgmd3s2/

### CODE REVIEW

**Review by chocolateimage — APPROVED** [review]

**Comment by chocolateimage** [comment]
I accepted it and also added my tutorial video

**Comment by candidexmedia** [comment]
Love the tutorial! Very easy to follow.

File: README.md [hunk 1]
```diff
@@ -16,12 +25,13 @@ Debian: `sudo apt-get install python3`
...
+* Make the necessary settings for fps, duration (in seconds), start and end position and of course select the type of easing
 * Click on "Generate and copy".
 * Go in kdenlive
 * Add a transform effect to a clip.
 * In the transform effect, Click the hamburger menu button left to the timestamp.
 * Select "Import Keyframes from Clipboard..."
 * Change the time offset (if necessary)
-* Disable "Limit keyframe number"
\ No newline at end of file
+* Disable "Limit keyframe number"
```
Comment (chocolateimage) [inline README.md:37]: what is changed here? (at line 37)

Comment (candidexmedia) [reply README.md:37]: I don't remember changing anything here. Maybe there was a the line break at the end of the file that was deleted?

Screenshot_20230418-081458

**Pull request merged**